spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/sleuth/StackdriverHttpRequestParser.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spring.autoconfigure.trace.sleuth;
+
+import java.net.URI;
+
+import brave.SpanCustomizer;
+import brave.http.HttpRequest;
+import brave.http.HttpRequestParser;
+import brave.http.HttpTags;
+import brave.propagation.TraceContext;
+
+
+/**
+ * An {@link HttpRequestParser} that fills information for Stackdriver Trace.
+ *
+ * <p>Based on {@code org.springframework.cloud.sleuth.instrument.web.SleuthHttpClientParser}.
+ *
+ * @author Ray Tsang
+ * @author Travis Tomsu
+ */
+public class StackdriverHttpRequestParser implements HttpRequestParser {
```

Comment:
The old code also had server response parsing/customization. For sanity checking, could you test the same sample app from old repo and new repo, and just compare what shows up in Stackdriver console?

Reply:
They look to capture all the same info to me. PTAL

![3H8pF5CTnRcjMyB (1)](https://user-images.githubusercontent.com/13141550/97006749-4ec21600-150e-11eb-8fed-47d9270440da.png)